src/Compilers/CSharp/Portable/Binder/Binder.cs

```diff
@@ -347,7 +347,7 @@ internal bool BindingTopLevelScriptCode
             get
             {
                 var containingMember = this.ContainingMemberOrLambda;
-                switch (containingMember.Kind)
+                switch (containingMember?.Kind)
```

Comment:
I didn't understand what prompted this change.

Reply:
This is now used in error scenarios where `containingMember` can be null.

---
In reply to: [167699262](https://github.com/dotnet/roslyn/pull/24756#discussion_r167699262) [](ancestors = 167699262)